src/include/lkl/syscall-overrides-fstat.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _LKL_SYSCALL_OVERRIDES_FSTAT_H
+#define _LKL_SYSCALL_OVERRIDES_FSTAT_H
+
+typedef long (*syscall_fstat_handler)(int, void*);
+typedef long (*syscall_newfstatat_handler)(
+    int, const char*, void*, int);
+
+// long syscall_fstat_override(int fd, struct stat* stat);
+
+// long syscall_newfstatat_override(int dfd, const char *fn,
+      // struct stat *statbuf, int flag);
```

Comment:
Please make these doc comments, starting /** and correctly indented.

Reply:
Oh, I totally overlooked the `doc` part. Will do.